plugins/modules/nameserver_info.py

```diff
@@ -57,6 +57,14 @@
             - How often to retry on SERVFAIL errors.
         type: int
         default: 0
+    servers:
+        description:
+            - A list of DNS server IP addresses to use for resolving nameserver information.
+            - When specified, these servers are used instead of the system's default DNS settings.
+        type: list
+        elements: str
+        required: false
+        default: null
```

Comment:
There's no need to specify defaults. But you do need to specify `version_added`:
```suggestion
        version_added: 2.7.0
```